PyRDF/backend/Backend.py

```diff
@@ -52,6 +52,27 @@ def __init__(self, config={}):
 
         """
         self.config = config
+        self.includes = []
+
+    @staticmethod
+    def declare_headers(includes=[]):
+        """
+        Declares all required headers using
+        PyROOT's "ROOT.gInterpreter.Declare".
+
+        parameters
+        ----------
+        includes : list (optional)
+            This list should consist of all necessary C++
+            headers as a string. This parameter defaults
+            to an empty list.
+
+        """
+        import ROOT
```

Comment:
Should we place this import at the beginning of the module?
This helper method will be executed by the backend, which anyway will import ROOT at some point because it needs to run the computation, right? So it does not hurt to have it in the module instead of in the function.

Reply:
@etejedor . Yes, that works. Will change this 👍